packages/core-ethereum/src/ethereum.ts

```diff
@@ -303,26 +283,13 @@ export async function createChainWrapper(
   }
 
   async function finalizeChannelClosure(channels: HoprChannels, counterparty: Address): Promise<Receipt> {
-    const populatedTx = await channels.populateTransaction.finalizeChannelClosure(counterparty.toHex())
-    const transaction = await sendTransaction(
-      checkDuplicate,
-      populatedTx,
-      channels.finalizeChannelClosure,
-      counterparty.toHex()
-    )
+    const transaction = await sendTransaction(checkDuplicate, channels, 'finalizeChannelClosure', counterparty.toHex())
     return transaction.hash
     // TODO: catch race-condition
```

Comment:
Do we know what is this comment about?

Reply:
its possible that the second time we run this function, there is still a pending tx from last time